docs/concepts/projects.md

```diff
@@ -247,7 +247,7 @@ requires-python = ">=3.11"
 dependencies = []
 
 [project.scripts]
-hello = "example_packaged_app:hello"
+example-packaged-app = "example_packaged_app:hello"
```

Comment:
This should be `:main` right?

Reply:
oops, fixed.

Also I'm very used to force pushing but shouldn't do that since you squash merge here. I just haven't learned the lesson yet..